Test failure on spec

Not sure if contextual or if already taken care of but this can be reproduced locally (and was spotted on the CI as well):

```
--- FAIL: TestParse (0.00s)
    Error Trace:    spec_test.go:69
    Error:          Not equal: "name1" (expected)
                    != "name2" (actual)

    Error Trace:    spec_test.go:70
    Error:          Not equal: 1 (expected)
                    != 2 (actual)

    Error Trace:    spec_test.go:71
    Error:          Not equal: "image1" (expected)
                    != "image2" (actual)

    Error Trace:    spec_test.go:69
    Error:          Not equal: "name2" (expected)
                    != "name1" (actual)

    Error Trace:    spec_test.go:70
    Error:          Not equal: 2 (expected)
                    != 1 (actual)

    Error Trace:    spec_test.go:71
    Error:          Not equal: "image2" (expected)
                    != "image1" (actual)
```

/cc @aluzzardi @stevvooe

--- a/spec/spec_test.go
+++ b/spec/spec_test.go
@@ -65,10 +65,9 @@ services:
 		s := Spec{}
 		assert.NoError(t, s.Read(f))
 		assert.Equal(t, size, len(s.JobSpecs()))
-		for i, jobSpec := range s.JobSpecs() {
-			assert.Equal(t, fmt.Sprintf("name%d", i+1), jobSpec.Meta.Name)
-			assert.EqualValues(t, i+1, jobSpec.GetService().Instances)
-			assert.Equal(t, fmt.Sprintf("image%d", i+1), jobSpec.Template.GetContainer().Image.Reference)
+		for _, jobSpec := range s.JobSpecs() {
+			assert.Equal(t, fmt.Sprintf("name%d", jobSpec.GetService().Instances), jobSpec.Meta.Name)
+			assert.Equal(t, fmt.Sprintf("image%d", jobSpec.GetService().Instances), jobSpec.Template.GetContainer().Image.Reference)
 		}
 	}
 }

